### ISSUE TITLE
Upgrade ruby version to 2.0.0-p648

### ISSUE THREAD

**Issue by shirshendu** [Tendrl/api#376]
This is the latest available system ruby on RHEL/CentOS.
This is the interim step before moving to rbenv/rvm/docker deployments, which would give us flexibility to use latest ruby.

**Comment by r0h4n** [comment]
@TimothyAsir  What are the risks for moving the current ruby version used for tendrl-api to version 2.0.0 for RHEL

**Comment by TimothyAsirJeyasing** [comment]
Its already available in rhel-7-server-rpms channel. The current available version is  2.0.0.648-33.el7_4 which will be installed while installing tendrl-api.

**Comment by shtripat** [comment]
In that case we can give a try but any impact on the gems we are using/building?

**Comment by mbukatov** [comment]
You are right about the latest version of ruby available on RHEL/CentOS 7, it's `ruby-2.0.0.648-33.el7_4`.

But I don't understand what small change you have in mind in the description of this issue. We will use this particular ruby version when deploying on up to date RHEL system, we don't have to change anything on our side to use it.

But I see two occurrences of some a ruby version `ruby 2.0.0p598` in 2 files:

```
$ git grep -i ruby\ 2.0
Gemfile.lock:   ruby 2.0.0p598
README.adoc:. Install Ruby 2.0.0p598.
```
If you mean updating those, I'm ok with it.

Btw, that ruby version in `Gemfile.lock` is important for dev environment only, right?

**Comment by shirshendu** [comment]
@mbukatov the way we are deploying through RPMs, we cannot control which patch version of ruby to depend on. However, `Gemfile.lock` is integral to the RPM build process.
Apart from the `Gemfile.lock` file, we also need to update the .ruby-version` file, which is the one that will change the ruby dependency of the project. That's how we specify ruby version dependency of a project, and package managers, ruby version managers tend to respect it.

**Comment by shirshendu** [comment]
> Its already available in rhel-7-server-rpms channel. The current available version is 2.0.0.648-33.el7_4 which will be installed while installing tendrl-api.

Nice, this means there is no downstream dependency of the change I wish to make. Thanks for the info.
